posthog/models/plugin.py

```diff
@@ -53,3 +53,10 @@ class PluginAttachment(models.Model):
     file_name: models.CharField = models.CharField(max_length=200)
     file_size: models.IntegerField = models.IntegerField()
     contents: models.BinaryField = models.BinaryField()
+
+
+class PluginStorage(models.Model):
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE, null=True)
```

Comment:
Q: Why can this be null?

Reply:
Ugh, seems to have been an overlooked copypasta. Thanks for spotting!